sdk/eventhub/eventhubs-checkpointstore-blob/package.json

```diff
@@ -61,7 +61,7 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/event-hubs": "^5.5.0",
+    "@azure/event-hubs": "^5.6.0",
     "@azure/logger": "^1.0.0",
     "@azure/storage-blob": "^12.4.1",
```

Comment:
Doesn't storage-blob need to be updated as well?

Reply:
I'm not entirely sure how the set-dev.js script chooses versioning policies, but this was the minimal set of changes necessary to get the build passing again. I think it has something to do with @Azure/storage-blob@2.4.1 being in the allowedAlternativeVersions entry https://github.com/Azure/azure-sdk-for-js/blob/ce818c60ece35debcda6a5bfda17711e7416e8cb/common/config/rush/common-versions.json#L44-L50

Although reading that comment it sounds like we should move off of 12.4.1 regardless as 12.6.0 has been out for a while. I can try it and see if things are behaving